webapp/src/DOMJudgeBundle/Service/ScoreboardService.php

```diff
@@ -325,36 +322,61 @@ public function calculateScoreRow(Contest $contest, Team $team, Problem $problem
 
         foreach ($submissions as $submission) {
             /** @var Judging|null $judging */
-            $judging    = $submission->getJudgings()->first() ?: null;
-            $absSubmitTime = (float)$submission->getSubmittime();
-            $submitTime = $contest->getContestTime($absSubmitTime);
+            $judging = $submission->getJudgings()->first() ?: null;
 
             // Check if this submission has a publicly visible judging result:
             if ($judging === null || ($verificationRequired && !$judging->getVerified()) || empty($judging->getResult())) {
-                $pendingJury++;
+                // For the jury: only consider it pending if we don't have a correct one yet
```

Comment:
This is pedantic, but can you please finish these sentences with a . ? Also for the other comments.